windows/deploy_vm/deploy_vm_from_ova.yml

```diff
@@ -41,12 +41,6 @@
   ansible.builtin.set_fact:
     vm_exists: true
 
-# Power on VM and wait for guest fullname reported by VMware tools
-- name: "Power on VM"
-  include_tasks: ../../common/vm_set_power_state.yml
-  vars:
-    vm_power_state_set: "powered-on"
-
 - name: "Get VM info"
   include_tasks: ../../common/vm_get_vm_info.yml
```

Comment:
Move `vm_get_vm_info.yml` after `collect_ovf_vm_config.yml` as it must be executed after VM power-on for retrieving primary nic MAC.

Reply:
Change it. Thanks